ansible/roles/openshift_setup/files/update_oauth_provider_to_htpasswd.py

```diff
@@ -0,0 +1,60 @@
+#! /usr/bin/env python
+import os
+import sys
+import time
+import yaml
+
+
+def parse(filename):
+    try:
```

Comment:
minor nit, but `open` is a context manager, so you can actually get this behavior wiht
```python
with open(filename, 'r') as f:
    raw_data = f.read()
return yaml.load(raw_data)
```